data/human/paradise world jobs.txt

```diff
@@ -453,13 +453,13 @@ mission "Paradise Job: Debtors"
 	name "Debtors to <planet>"
 	job
 	repeat
-	description `These <bunks> poor souls have bankrupted themselves attempting to live beyond their means. Due to media scrutiny on the issue, the local government will pay you <payment> to transport them to <destination> where they can "make something useful of themselves."`
+	description `These <bunks> poor souls have bankrupted themselves attempting to live beyond their means. Due to media scrutiny on the issue, the local government will pay you <payment> to transport them in a cell to <destination> where they can "make something useful of themselves."`
```

Comment:
This is too long (and was probably already too long before).
![image](https://user-images.githubusercontent.com/17688683/209738433-745f338f-4cf2-4e73-bbeb-547c4c96bf28.png)
![image](https://user-images.githubusercontent.com/17688683/209738479-d67e329e-2e0c-4020-8ed0-12c6c76dd182.png)